package.json

```diff
@@ -81,6 +81,7 @@
     "eslint-plugin-prettier": "^3.1.2",
     "eslint-plugin-react": "^7.19.0",
     "eslint-plugin-react-hooks": "^2.5.0",
+    "filesize.js": "^2.0.0",
```

Comment:
please remove `humanize-plus` and `@types/humanize-plus`. They don't seem to be used.

Reply:
in addition, this should be among `dependencies`

Reply:
1. If I put this inside the `dependencies` section then `microbundle` won't include it in the final bundle. I found an explanation on why this happens and why should I put it inside the `devDependencies` in [microbundle's official docs](https://github.com/developit/microbundle/wiki/How-Microbundle-decides-which-dependencies-to-bundle). I don't know why microbundle works in this way, it's against the standard (and I hate this the same as you do).
2. We still use `humanize-plus` in `hardwareInfo.ts`

Reply:
1. But this is as expected. By having it among `dependencies`, the yarn will resolve that at `yarn install` time of the consuming package (i.e. `assisted-ui`). I understand this is a chicken-egg issue at development time, but you can workaround it by
- simply trust since you have it already tested within `devDependencies`
- do tentative `yarn add` of the package within `assisted-ui` (do not commit that change there)
Once we release the library, everything will be good again.

By inlining all dependencies, our dist would be unnecessary big. And we would dictate all the transpiling rules even on the places where the consuming application build process should be used. Another disadvantage will be in no-resuse of that 3rd party code within the consuming application - consider the case that the same library is used by another dependency as well. The tree shaking will not work here.

2. You are right, for CPU speed. Let's keep it then.

Reply:
Moved `filesize.js` to dependencies here: 72026c95c77f190735363a8481f0cb31d2fca92e